test/controllers/AdditionalMetadataClosureStatusControllerSpec.scala

```diff
@@ -0,0 +1,309 @@
+package controllers
+
+import cats.implicits.catsSyntaxOptionId
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock._
+import configuration.{GraphQLConfiguration, KeycloakConfiguration}
+import controllers.util.MetadataProperty.closureType
+import graphql.codegen.AddBulkFileMetadata.{addBulkFileMetadata => abfm}
+import graphql.codegen.GetConsignment.getConsignment
+import graphql.codegen.types.UpdateBulkFileMetadataInput
+import io.circe.generic.auto._
+import io.circe.syntax._
+import io.circe.parser.decode
+import org.pac4j.play.scala.SecurityComponents
+import org.scalatest.concurrent.ScalaFutures.convertScalaFuture
+import play.api.http.Status._
+import play.api.test.CSRFTokenHelper.CSRFRequest
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{GET, POST, contentAsString, contentType, defaultAwaitTimeout, redirectLocation, status}
+import services.{ConsignmentService, CustomMetadataService}
+import testUtils.{CheckPageForStaticElements, FrontEndTestHelper}
+import uk.gov.nationalarchives.tdr.GraphQLClient.Error
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+// scalastyle:off line.size.limit
+class AdditionalMetadataClosureStatusControllerSpec extends FrontEndTestHelper {
+
+  val wiremockServer = new WireMockServer(9006)
+  val parentFolderId: UUID = UUID.randomUUID()
+
+  override def beforeEach(): Unit = {
+    wiremockServer.start()
+  }
+
+  override def afterEach(): Unit = {
+    wiremockServer.resetAll()
+    wiremockServer.stop()
+  }
+
+  val checkPageForStaticElements = new CheckPageForStaticElements
+  implicit val ec: ExecutionContext = ExecutionContext.global
+
+  val fileIds: List[UUID] = List(UUID.randomUUID())
+
+  "getClosureStatusPage" should {
+    "render the closure status page and closure status checkbox should be unchecked" in {
+      val consignmentId = UUID.randomUUID()
+      val controller = createController("Open")
+
+      val response = controller.getClosureStatusPage(consignmentId, fileIds)
+        .apply(FakeRequest(GET, s"/consignment/$consignmentId/additional-metadata/closure-status").withCSRFToken)
+
+      status(response) mustBe OK
+      contentType(response) mustBe Some("text/html")
+
+      verifyClosureStatusPage(contentAsString(response), consignmentId)
+
+      wiremockServer.verify(postRequestedFor(urlEqualTo("/graphql")))
+    }
+
+    "render the closure status page and closure status checkbox should be checked" in {
```

Comment:
Maybe reword this test a bit to make it clear what the scenario is for the checkbox being checked: `render the closure status page with the closure status checkbox checked when closure status already set to 'closed'`